src/main/webapp/i18n/de/statistics.json

```diff
@@ -1,44 +1,44 @@
 {
     "statistics": {
         "statistics-title": "Statistiken",
-        "amountOfStudents": "# an Studenten",
+        "amountOfStudents": "# an Studierenden",
         "span": {
             "day": "Tag",
             "week": "Woche",
             "month": "Monat",
             "quarter": "Quartal",
             "year": "Jahr"
         },
-        "title": "Benutzerstatistiken",
+        "title": "Nutzungsstatistiken",
         "course_statistics_title": "Kursstatistiken",
         "submissions": "Abgaben",
-        "active_users": "Aktive Nutzer",
-        "logged_in_users": "Eingeloggte Nutzer",
+        "active_users": "Aktive User",
+        "logged_in_users": "Eingeloggte User",
         "released_exercises": "Veröffentlichte Aufgaben",
         "exercises_due": "Aufgaben mit Abgabeende",
         "conducted_exams": "Durchgeführte Klausuren",
         "exam_participations": "Klausurteilnahmen",
         "exam_registrations": "Klausuranmeldungen",
-        "active_tutors": "Aktive Tutoren",
+        "active_tutors": "Aktive Tutor:innen",
         "created_results": "Erstellte Resultate",
         "created_feedbacks": "Erstellte Feedbacks",
         "questions_asked": "Gestellte Fragen",
         "questions_answered": "Beantwortete Fragen",
         "score_distribution": "Bewertungsverteilung",
         "submissionsTitle": "# an Abgaben",
-        "active_usersTitle": "# an Benutzer",
-        "logged_in_usersTitle": "# an Benutzer",
+        "active_usersTitle": "# an User",
```

Comment:
If User is meant as a german word here, it would probably have to be 'Usern'

Reply:
I'll change it to: # User, # Angaben... (the decision on User vs. User:in vs. Benutzer:in will be discussed later)